drivers/storage/portworx/testspec/prometheusNewOperatorClusterRole.yaml

```diff
@@ -0,0 +1,66 @@
+# This OperatorClusterRole is for Prometheus operator 0.50.0
```

Comment:
Suggest to change the file name to prometheusNewOperatorClusterRole-v0.50.0.yaml